src/CalcViewModel/UnitConverterViewModel.cpp

```diff
@@ -152,11 +152,14 @@ UnitConverterViewModel::UnitConverterViewModel(const shared_ptr<UCM::IUnitConver
     PopulateData();
 }
 
-void UnitConverterViewModel::ResetView()
+void UnitConverterViewModel::ResetView(bool resetValues)
```

Comment:
I'm not really a fan of this change because it leads to awkward code like `Model->RefreshCurrencyRatios(false);`  Are we refreshing the currenty ratios or not?  Now the casual reader needs to know what `true`/`false` mean in the context of this function and overall clarity of the code is reduced.

My suggestion would be to write `void UnitConverterViewModel::ResetValues()` and call it where appropriate.

Reply:
Same opinion for `UpdateCategory`.

Reply:
Ping @rudyhuyn

Reply:
I totally forgot this PR, I will update it tonight!

Reply:
I totally refactored this PR without the boolean, I agree, it's better this way!